src/LazyDefinition.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Definitions;
+
+use ProxyManager\Factory\LazyLoadingValueHolderFactory;
+use Psr\Container\ContainerInterface;
+use Yiisoft\Definitions\Contract\DefinitionInterface;
+use Yiisoft\Definitions\Helpers\Normalizer;
+
+final class LazyDefinition implements DefinitionInterface
+{
+    public function __construct(
+        private mixed $definition,
+        /**
+         * @var class-string
```

Comment:
```suggestion
         * @pslam-var class-string
```

Reply:
What the reason it for?

Reply:
`class-string` is psalm specific type. Base phpdoc do not support it.

Reply:
PHPStorm perfectly works with it, why should I care about phpdoc standards?

Reply:
Care about standarts is good practic 😎

Reply:
Because we're going to generate API documentation same as we did for Yii2.

Reply:
Fixed